tools/cxbe/Xbe.cpp

```diff
@@ -735,19 +739,22 @@ Xbe::Xbe(class Exe *x_Exe, const char *x_szTitle, bool x_bRetail)
             hwc      += 2;
         }
 
-        // write default "OpenXDK" logo bitmap
+        // write logo bitmap
         {
-            printf("Xbe::Xbe: Generating \"OpenXDK\" Logo Bitmap...");
+            printf("Xbe::Xbe: Generating Logo Bitmap...");
 
             uint08 *RawAddr = GetAddr(m_Header.dwLogoBitmapAddr);
 
-            memset(RawAddr, 0, 100*17);
-
-            memcpy(RawAddr, OpenXDK, dwSizeOfOpenXDK);
-
-            m_Header.dwSizeofLogoBitmap = dwSizeOfOpenXDK;
-
-            printf("OK\n");
+            if (logo)
+            {
+                memcpy(RawAddr, logo->data(), logo->size());
```

Comment:
`&logo[0]` is more correct, imo.

Reply:
I'm not sure what you mean by undefined, there is no undefined behavior here. `std::vector::data()` is perfectly fine for both reading and writing, and I prefer it over `&logo[0]` because it makes it visually clear that we're not dealing with POD.